scripts/python/tao_build.py

```diff
@@ -222,6 +227,95 @@ def configure(root, args):
+
+        def _action_env(key, value, cmd="build"):
+            f.write(f"{cmd} --action_env {key}={value}\n")
+
+        def _write(line, cmd="build"):
+            f.write(f"{cmd} {line}\n")
+
+        python_bin = os.path.join(args.venv_dir, "bin", "python3")
+        _action_env("PYTHON_BIN_PATH", python_bin)
+        _action_env("GCC_HOST_COMPILER_PATH", which("gcc"))
+        _action_env("CC", which("gcc"))
+        _action_env("CXX", which("g++"))
+        (
+            tf_major,
+            tf_minor,
+            is_pai,
+            tf_header_dir,
+            tf_lib_dir,
+            tf_lib_name,
+            tf_cxx11_abi,
+            tf_pb_version,
+        ) = get_tf_info(python_bin)
+        _opt("cxxopt", f"-D_GLIBCXX_USE_CXX11_ABI={tf_cxx11_abi}")
+        _opt("host_cxxopt", f"-D_GLIBCXX_USE_CXX11_ABI={tf_cxx11_abi}")
+        _action_env("BLADE_WITH_TF", "1")
+        _action_env("IF_CXX11_ABI", int(tf_cxx11_abi))
+        _action_env("TF_IS_PAI", int(is_pai))
+        _action_env("TF_MAJOR_VERSION", tf_major)
+        _action_env("TF_MINOR_VERSION", tf_minor)
+        _action_env("TF_HEADER_DIR", tf_header_dir)
+        _action_env("TF_SHARED_LIBRARY_DIR", tf_lib_dir)
+        _action_env("TF_SHARED_LIBRARY_NAME", tf_lib_name)
+        _action_env("TF_PROTOBUF_VERSION", tf_pb_version)
+        # Build environments. They all starts with `DISC_BUILD_`.
+        host = socket.gethostname()
+        ip = socket.gethostbyname(host)
+        _action_env("DISC_BUILD_VERSION", args.version)
+        _action_env("DISC_BUILD_GIT_BRANCH", git_branch().decode("utf-8").replace('/', '-'))
+        _action_env("DISC_BUILD_GIT_HEAD", git_head().decode("utf-8"))
+        _action_env("DISC_BUILD_HOST", host)
+        _action_env("DISC_BUILD_IP", ip)
+        _action_env("DISC_BUILD_TIME", datetime.today().strftime("%Y%m%d%H%M%S"))
+
+        is_cuda = not (args.cpu_only or args.dcu)
+        if is_cuda:
+            cuda_ver, _ = deduce_cuda_info()
+            if '11\.' in cuda_ver:
+                _action_env("TF_CUDA_COMPUTE_CAPABILITIES", "7.0,7.5,8.0")
+            elif '10\.' in cuda_ver:
+                _action_env("TF_CUDA_COMPUTE_CAPABILITIES", "7.0,7.5")
+            _action_env("NVCC", which("nvcc"))
+            _write("--test_tag_filters=-cpu", cmd="test")
+        elif args.cpu_only:
+            _write("--test_tag_filters=-gpu", cmd="test")
+
+        logger.info("configuring tao_bridge with bazel ......")
+
+    with cwd(tao_bazel_root), gcc_env(args.bridge_gcc):
+        # make sure version.h is generated
+        execute("bazel build --config=release //:version_header_genrule")
+
+    with cwd(root):
+        # copy version.h from tao_bridge
+        # NOTE(lanbo.llb): This is no longer needed when tao_compiler is build
+        # in workspace `org_tao_compiler` instead of `org_tensorflow`
+        execute(
+            f"cp {tao_bazel_root}/bazel-bin/tao_bridge/version.h tao_compiler/decoupling/version.h"
```

Comment:
This is because `version.h` here is for `tao_compiler` build, which is compiled in `tf_community` dir. Later when `tao_compiler` can be built in `tao_compiler` dir, this no longer needed.

Reply:
No better solution for now ..